@here/harp-text-canvas/lib/rendering/FontCatalog.ts

```diff
@@ -544,6 +544,8 @@ export class FontCatalog {
         (replacementGlyph as any).codePoint = codePoint;
         (replacementGlyph as any).character = char;
         (replacementGlyph as any).font = font;
+        // Glyphs for ASCII control characters and such are not really replacement glyphs.
```

Comment:
in what case do we have ascii control characters that need to be rendered?

Reply:
Replacement glyphs are created for all characters in a string. They will not be rendered, though, but later identified by their code points to be whitespace of line feeds and such.